flask/signals.py

```diff
@@ -52,4 +53,4 @@ def _fail(self, *args, **kwargs):
 appcontext_tearing_down = _signals.signal('appcontext-tearing-down')
 appcontext_pushed = _signals.signal('appcontext-pushed')
 appcontext_popped = _signals.signal('appcontext-popped')
-message_flashed = _signals.signal('message-flashed')
+message_flashed = _signals.signal('message-flashed')
```

Comment:
You removed the trailing newline in that file. Can you please revert that (simply amend the commit and force-push)? Python files should end with a linebreak.